docs/docs/Integrations/Nvidia/integrations-nvidia-ingest.md

```diff
@@ -0,0 +1,72 @@
+
+## Prerequisites
+
+* An NVIDIA Ingest endpoint, like `http://33.71.182.84:7670`. For more information on setting up an NVIDIA ingest endpoint, see the [NVIDIA Ingest quickstart](https://github.com/NVIDIA/nv-ingest?tab=readme-ov-file#quickstart).
+
+* The **NVIDIA Ingest** component requires the installation of additional dependencies to your Langflow environment. To install the dependencies in a virtual environment, run the following commands:
+```bash
+source **YOUR_LANGFLOW_VENV**/bin/activate
+uv sync --extra nv-ingest
+uv run langflow run
+```
+
+## Use the NVIDIA Ingest component in a flow
+
+The **NVIDIA Ingest** component accepts a **Message** inputs and outputs **Data**. The component uses the NVIDIA ingest endpoint to ingest a local file and extract the text.
+
+To use the NVIDIA Ingest component in your flow, follow these steps:
+1. Click the **NVIDIA Ingest** component in the component library and drag it onto the canvas.
+2. In the **NVIDIA Ingest endpoint** field, enter the URL of the NVIDIA ingest endpoint.
+3. In the **Path** field, enter the path to the file you want to ingest.
+4. Select which text type to extract from the file.
+The component supports text, charts, and tables.
+5. Select whether to split the text into chunks.
+Modify the splitting parameters in the component's **Configuration** tab.
+7. Click **Run** to ingest the file.
+8. To confirm the component is ingesting the file, open the **Logs** pane to view the output of the flow.
+9. To store the processed data in a vector database, add a **AstraDB Vector** component to your flow, and connect the **NVIDIA Ingest** component to the **AstraDB Vector** component with a **Data** output.
+
+![NVIDIA Ingest component flow](nvidia-component-ingest-astra.png)
+
+10. Run the flow.
+Inspect your Astra DB vector database to view the processed data.
+
+## NVIDIA ingest component parameters {#parameters}
+
+The NVIDIA ingest component has the following parameters.
+
+For more information, see the [NV-Ingest documentation](https://nvidia.github.io/nv-ingest/user-guide/).
+
+### Inputs
+
+| Name | Display Name | Info | Required/Default |
+|------|--------------|------|------------------|
+| base_url | NVIDIA Ingestion URL | The URL of the NVIDIA Ingestion API | Required |
+| path | Path | File path to process | Required |
+| extract_text | Extract Text | Extract text from documents | Default: True |
+| extract_charts | Extract Charts | Extract text from charts | Default: False |
+| extract_tables | Extract Tables | Extract text from tables | Default: True |
+| text_depth | Text Depth | Level at which text is extracted (applies before splitting). Support for 'block', 'line', 'span' varies by document type | Default: "document" |
```

Comment:
You are inconsistent with the punctuation of the text in the Info column.

Reply:
Removed the Default/Required column and added periods.